2021-roadmap.md

```diff
@@ -0,0 +1,59 @@
+# **Shipwright 2021 Roadmap**
+
+**Mission Statement:** Shipwright aims to do one thing well: build images from source. Automatically, securely, reliably.
+
+Features that further that goal should be welcomed, and prioritized to optimize that goal. Features that don't should be improved and removed if necessary.
+
+Shipwright should be simple to use and flexible to extend. Once set up, it should run smoothly for users and operators alike, and when failures happen (they always do) they should be easy to diagnose and fix.
+
+
+1. **Build ← you are here!**
+    *   Pluggable BuildStrategies, with pre-defined support for Buildpacks, Kaniko, S2i, Buildah; users can modify, install their own
+    *   Continue to improve built-in BuildStrategies for security best practices ([#169](https://github.com/shipwright-io/build/issues/169))
+2. **API Improvements / Simplification**
+    *   The API should be small, powerful and easy to use
+    *   Synchronous webhook validation ([#685](https://github.com/shipwright-io/build/pull/685))
+        *   Deprecate and remove BuildStatus and BuildStrategyStatus
+    *   BuildRun status should include resolved git commit ([#205](https://github.com/shipwright-io/build/issues/205)) and built image digest ([#618](https://github.com/shipwright-io/build/issues/618))
+    *   BuildStrategy Parameterization ([#697](https://github.com/shipwright-io/build/pull/697))
+        *   Move builder + dockerfile fields to strategy-specific params
+    *   Don’t rely on `creds-init` or PipelineResources ([#696](https://github.com/shipwright-io/build/issues/696))
+        *   These features not likely to be extended to Tekton v1
+        *   Remove auto-generating SAs at runtime ([#679](https://github.com/shipwright-io/build/issues/679))
+    *   [API Review](https://github.com/shipwright-io/build/issues/516) + API Hygiene
+        *   Deprecate and remove BuildRun [`.status.conditions`](https://github.com/shipwright-io/build/issues/517)
```

Comment:
This should be `status.reason` and `status.Succeeded, or you can simply phrase it as `Deprecate and remove BuildRun deprecated Status fields`